frontend/express/public/stylesheets/styles/blocks/_sidebar.scss

```diff
@@ -133,8 +141,14 @@ $bg-color: #24292e;
     }
 
     &__header {
-        padding: 16px 12px;
+        padding: 12px;
         position: relative;
+        left: 60px;
+        top: 16px;
+        border: 1px solid #424243;
+        box-sizing: border-box;
+        box-shadow: 0px 11px 14px rgba(0, 0, 0, 0.371635);
+        border-radius: 4px;
 
```

Comment:
the above css that you added, should not be added to header since the same class is used for other menus. plus it doesn't look its consistent with the designs.
this should be added to the app selector component you made.